src/natcap/invest/habitat_quality.py

```diff
@@ -1044,23 +1044,13 @@ def _validate_threat_path(threat_path, lulc_key):
     """
     # Checking threat path exists to control custom error messages
     # for user readability.
-    try:
-        threat_gis_type = pygeoprocessing.get_gis_type(threat_path)
```

Comment:
Okay, I see that in validation for this an empty entry is returned with no error.

Reply:
Yes, empty entries are passed through and don't raise any validation warning. Note that this applies even when the column _is_required - NA values don't fail validation because in some cases we do allow NA values.